teamB/sangwoo/app.js

```diff
@@ -49,6 +49,137 @@ app.post("/user/signup", async (req, res) => {
     res.status(201).json({ message: "userCreated" });
 });
 
+app.post("/posts", async (req, res) => {
+    const { title, content, user_id } = req.body;
+
+    await myDataSource.query(
+        ` 
+        INSERT INTO posts(
+            title,
+            content,
+            user_id
+        ) VALUES (?,?,?);
+        `,
+        [title, content, user_id]
+    );
+    res.status(201).json({ message: "postCreated" });
+});
+
+
+app.get("/posts", async (req, res) => {
+    await myDataSource.manager.query(
+        `SELECT 
+            u.id AS userId,
+            u.profile_Image AS userProfileImage,
+            p.id AS postingId,
+            p.title AS postingTitle,
+            p.content AS postingContent
+        FROM users u
+        INNER JOIN posts p ON
+        u.id = p.user_id
+        `,
+        (err, rows) => {
+            res.status(200).json({ data: rows });
+        }
+    );
+});
+
+
+app.get("/posts/:userId", async (req, res) => {
+    const { userId } = req.params;
+
+    await myDataSource.manager.query(
+        `SELECT
+            u.id AS userId,
+            u.name AS userName,
+            u.profile_Image AS userProfileImage,
+            p.id AS postingId,
+            p.title AS postingTitle,
+            p.content AS postingContent
+        FROM users u
+        INNER JOIN posts p ON u.id = p.user_id
+        WHERE u.id = ${userId}
+        `,
+        (err, rows) => {
+            let postings = [];
+            let data = {
+                userId : rows[0].userId,
+                userName: rows[0].userName,
+                userProfileImage : rows[0].userProfileImage,
+                postings
+            }
+            for(let i in rows){
+                postings.push({
+                    postingId: rows[i].postingId,
+                    postingTitle: rows[i].postingTitle,
+                    postingContent: rows[i].postingContent
+                })
+            }
+            res.status(200).json({ data: data });
+        }
+    );
+});
+
+
+app.patch("/posts/:userId/:postId", async(req, res) => {
+    const {userId} = req.params;
+    const {postId} = req.params;
```

Comment:
params도 구조분해 할당을 통해서 한 줄로 줄여주세요~
```suggestion
    const { userId, postId } = req.params;
```